src/rsa.rs

```diff
@@ -244,6 +255,49 @@ impl RSAKeyPair {
     pub fn public_modulus_len(&self) -> usize {
         unsafe { RSA_size(self.rsa.as_ref()) }
     }
+
+    /// Sign the given message, hashed with the given digest algorithm.
```

Comment:
Please document the function by mentioning what each parameter does. (See the documentation for other public API functions at https://briansmith.org/rustdoc/ring/.)